code/modules/clothing/spacesuits/rig/suits/combat.dm

```diff
@@ -106,3 +106,57 @@
 		bio = 100,
 		rad = 50
 	)
+
+/obj/item/rig/combat/knight
+	name = "knight hardsuit control module"
+	desc = "A RIG module for a \"Knight\" model hardsuit, jointly manufactured by Divisors and Numericals of the New Testament. \
+	The Tau Cross shining brightly upon its shoulder, it offers moderate combat protection against many a type of threat to the Absolute."
+	icon_state = "knight_rig"
+	suit_type = "knight rig"
+	armor_list = list(
+		melee = 50,
+		bullet = 45,
+		energy = 30,
+		bomb = 50,
+		bio = 100,
+		rad = 50
+	)
+	slowdown = 0.3
+	drain = 4
+	offline_slowdown = 3
+	offline_vision_restriction = 1
+	max_upgrades = 1
+	chest_type = /obj/item/clothing/suit/space/rig/combat/knight
+	helm_type =  /obj/item/clothing/head/helmet/space/rig/combat/knight
+	boot_type =  /obj/item/clothing/shoes/magboots/rig/combat/knight
+	glove_type = /obj/item/clothing/gloves/rig/combat/knight
+
+/obj/item/rig/combat/knight/equipped
+	initial_modules = list(
+		/obj/item/rig_module/storage,
+		/obj/item/rig_module/device/flash,
+		/obj/item/rig_module/maneuvering_jets,
+		/obj/item/rig_module/held/shield
+		)
+
+/obj/item/clothing/suit/space/rig/combat/knight
+	name = "knight hardsuit breastplate"
+
+/obj/item/clothing/gloves/rig/combat/knight
+	name = "knight hardsuit gauntlets"
+
+/obj/item/clothing/shoes/magboots/rig/combat/knight
+	name = "knight hardsuit greaves"
+
+/obj/item/clothing/head/helmet/space/rig/combat/knight
+	name = "knight hardsuit full helm"
+	desc = "One's spirit is generally the greatest shield."
+	light_overlay = "helmet_light_dual"
+	armor_list = list(
```

Comment:
The RIG overwrite all armor, not just the ones that aren't defined.

Reply:
Does that mean the blackshield hood with its armor values defined above is moot as well?

Reply:
Well, I'm not 100% sure if that's the case, but I do know that it was the case for the Hydrogen Knight Rig Suit's chest pieces.